src/main/java/org/jeasy/flows/work/WorkContext.java

```diff
@@ -0,0 +1,23 @@
+/**  
+ * All rights Reserved, Designed By www.didichuxing.com.
+ * @Title:  WorkContext.java   <\br>
+ * @Package org.jeasy.flows.work   <\br>
+ * @Description:    TODO(用一句话描述该文件做什么)<\br>   
+ * @author: zyt     <\br>
+ * @date:   2018年12月14日 下午6:31:44   <\br>
+ * @version V1.0 <\br>
+ * @Copyright: 2018 www.didichuxing Inc. All rights reserved.<\br> 
+ */
+
+package org.jeasy.flows.work;
+
+/**   
+ * @ClassName:  WorkContext   .
+ * @Description:TODO(这里用一句话描述这个类的作用)  <\br> 
+ * @author: zyt <\br>
+ * @date:   2018年12月14日 下午6:31:44   <\br>
+ *   
+ */
+public interface WorkContext {
```

Comment:
What is the purpose of this empty interface? IMO the execution context should be a class which contain at least a data structure (like a map) to hold contextual data.